Makefile

```
@@ -71,7 +71,11 @@ install-ocamlformat:
 dev-depext:
 	opam depext -y $(TEST_DEPS)
 
-dev-deps:
+.PHONY: melange
+melange:
+	opam pin add melange https://github.com/melange-re/melange.git#c73a34394560ce78dbb43d0777abfc1d913f1667
```

this has been merged, should be at least 2f7a184400fd5d62c9160528a7ab4ce81874c024
```suggestion
	opam pin add melange https://github.com/melange-re/melange.git#2f7a184400fd5d62c9160528a7ab4ce81874c024
```

---

Does it also need `-y` for CI?

---

I prefer without `-y`, so devs can still choose if they want to install it. CI will work as setup-ocaml sets `OPAMYES` to 1 already:

https://github.com/ocaml/setup-ocaml/blob/a98ac6d6fc2c64a4cfae064b385e8cbf095867cb/src/setup-ocaml/installer.ts#L49